packages/firebase_crashlytics/lib/src/firebase_crashlytics.dart

```diff
@@ -165,6 +168,12 @@ class Crashlytics {
           'line': lineNumber,
         };
 
+        // The next section would throw an exception in some cases if there was no stop here.
```

Comment:
It feels like we should add a small unit test for `getStackTraceElements` since it is `@visibleForTesting`

Reply:
Do you mean for the `continue` part I added? I did it to clean up the tests.  
Otherwise, there are [three tests for the method](https://github.com/flutter/plugins/blob/7c0a3bf650e7c09a18469a43bb6f59ef186041e5/packages/firebase_crashlytics/test/firebase_crashlytics_test.dart#L147).

Reply:
Got it. If that path is being exercised in tests and not in production then I think it's tested and we're fine.